### PR TITLE
Adds official add-ons for DSPL in a concise manner

### PR DESCRIPTION
**Pull request by kyordhel**
Allows:

- One USB audio-output device such as:
    - USB power speaker.
    - Sound card (dongle).
- Wi-Fi adapter (IEEE 802.11ac or newer).
- Gigabit Ethernet Switch (IEEE 802.3ab or newer)

Restrictions:
- Small (linear dimension sum up to 15cm).
- No loose wires.
- No visible devices.
- No tape

**Remark:** This summarizes and replaces #556 and #634

### CODE REVIEW

File: general_rules/ExternalDevices.tex [hunk 1]
```diff
@@ -50,6 +50,27 @@ \subsection{Official Standard Laptop for DSPL}
...
+		\item Sound card (dongle).
+	\end{itemize}
+	\item One USB-powered IEEE 802.11ac (or newer) compliant device (Wi-Fi adapter).
+	\item One USB-powered IEEE 802.3ab (or newer) compliant device (Gigabit Ethernet Switch).
+\end{itemize}
+
+In all cases, the number of attached devices can never be more than three, and the sum of the linear dimensions of each device is limited to 15cm, excepting the speaker which is limited up to 30cm.
+
+Devices attached to the \iterm{OSL} must fit inside the TOYOTA HSR \iterm{Mounting Bracket} and not increase the dimensions of the robot.
+For this purpose, using short cables (no longer than 20cm) and and attaching the device to the OSL is advised.
```
Comment (LarsJanssenTUe) [inline general_rules/ExternalDevices.tex]: Remove the double "and"

File: general_rules/ExternalDevices.tex [hunk 2]
```diff
@@ -50,6 +50,27 @@ \subsection{Official Standard Laptop for DSPL}
...
+	\item One USB audio-output device such as:
+	\begin{itemize}
+		\item USB power speaker.
+		\item Sound card (dongle).
+	\end{itemize}
+	\item One USB-powered IEEE 802.11ac (or newer) compliant device (Wi-Fi adapter).
+	\item One USB-powered IEEE 802.3ab (or newer) compliant device (Gigabit Ethernet Switch).
+\end{itemize}
+
+In all cases, the number of attached devices can never be more than three, and the sum of the linear dimensions of each device is limited to 15cm, excepting the speaker which is limited up to 30cm.
```
Comment (johaq) [inline general_rules/ExternalDevices.tex]: "except for the speaker which..."

**Comment by LoyVanBeek** [comment]
LGTM!

**Comment by MatthijsBurgh** [comment]
Putting items in the bracket is hardly possible. It can just hold the OSL. Because the bracket is square and the torso is round, Some space between those two is created. In my opinion, placing the devices in this space (Using velcro to attach them to the robot.) would be a fair solution. Would you agree to this?

This picture is to show the space between the bracket and the robot itself. Not for the currently attached objects.

**Comment by kyordhel** [comment]
Sounds fair. 

In the image there is a lot of cables. That wouldn’t be OK (perhaps wirehoose?). The white box (wifi?) taped to the robot also doesn’t look nice. Can it be velcroed and get more concealed?

**Comment by MatthijsBurgh** [comment]
All the tape will disappear. Devices will be moved into the space between the bracket and the robot. All the wires will be nicely guided to the devices.

**Comment by kyordhel** [comment]
>All the tape will disappear. Devices will be moved into the space between the bracket and the robot. All the wires will be nicely guided to the devices.

Then I think is OK.

File: general_rules/ExternalDevices.tex (hunk 2 above)
Comment (kyordhel) [reply general_rules/ExternalDevices.tex]: Fixed

File: general_rules/ExternalDevices.tex (hunk 1 above)
Comment (kyordhel) [reply general_rules/ExternalDevices.tex]: Fixed

**Pull request merged**